include/tgfx/core/Stream.h

```diff
@@ -65,4 +66,28 @@ class Stream {
    */
   virtual bool rewind() = 0;
 };
+
+/**
+ * StreamFactory creates instances of Stream and can be inherited externally to implement the
+ * loading of cache files without local path, such as caching data into a database, or assets
+ * resource files on the Android platform.
+ */
```

Comment:
StreamFactory creates Stream instances and can be implemented externally to load cache files that do not have local paths, such as binary data in a database or asset resources on the Android platform.